cvat-ui/src/components/requests-page/requests-list.tsx

```diff
@@ -33,10 +33,17 @@ function RequestsList(props: Props): JSX.Element {
     const dispatch = useDispatch();
     const { query, count } = props;
     const { page } = query;
-    const { requests } = useSelector((state: CombinedState) => state.requests);
+    const { requests, disabled } = useSelector((state: CombinedState) => state.requests);
```

Comment:
useSelector((state: CombinedState) => { requests: state.requests, disabled: state.disabled }, shallowEqual);

would be more optimized as `state.requests` may change more often than `state.requests.requests` and `state.requests.disabled`.

Reply:
Applied